docs/assets/polished.js

```diff
@@ -1410,10 +1409,15 @@
    * }
    */
   function hideVisually() {
+    // Note: The use of `clip-path` causes performance degredation on scroll events in Chrome.
+    // For more information, see:
+    // * h5bp/html5-boilerplate#2021
+    // * zurb/foundation-sites#10914
+    // * twbs/bootstrap#24906
     return {
       border: '0',
       clip: 'rect(0 0 0 0)',
-      clipPath: 'inset(50%)',
+      // clipPath: 'inset(50%)',
```

Comment:
I chose to comment this out instead of removing the line for posterity and to make it explicit that `clip-path` is _not_ being used.